### DISCUSSION TITLE
posts/tutos/my-first-app/3-first-run-of-the-project/

### DISCUSSION THREAD

**Discussion by giscus** [Kapusch/blog-dotnet-maui#5] [Blog comments]
# posts/tutos/my-first-app/3-first-run-of-the-project/

Allez, il est temps de faire tourner l’application. On en profitera aussi pour jeter un œil au code source ! 🙂

https://www.mauicestclair.fr/posts/tutos/my-first-app/3-first-run-of-the-project/

**Comment by Mkkaba** [comment]
J’ai cherché des articles sur .Net MAUI, mais quand j’ai vue le votre vraiment je me suis sentis vraiment merveilleux, j’ai crue que avec C# je ne pourrais pas faire plus car xamarin ferme ses portes dans pas longtemps donc j’ai essayé de savoir si y’a un autre moyen de développer des apk  avec C#.
Vraiment merci beaucoup et je vous encourage.
J’ai eu l’eau à la bouche quand j’ai vue votre réalisation avec MAUI, votre apk MyVanity, c’est une merveille.
Ma suggestion à ce niveau, c’est le bouton delete, je n’ai pas vue un point où on peu supprimer un article qu’on a publier par oublie, encore vraiment merci beaucoup pour tout.
Bien cordialement

**Reply by Kapusch** [reply]
> J’ai cherché des articles sur .Net MAUI, mais quand j’ai vue le votre vraiment je me suis sentis vraiment merveilleux, j’ai crue que avec C# je ne pourrais pas faire plus car xamarin ferme ses portes dans pas longtemps donc j’ai essayé de savoir si y’a un autre moyen de développer des apk  avec C#.
> Vraiment merci beaucoup et je vous encourage.
> J’ai eu l’eau à la bouche quand j’ai vue votre réalisation avec MAUI, votre apk MyVanity, c’est une merveille.
> Ma suggestion à ce niveau, c’est le bouton delete, je n’ai pas vue un point où on peu supprimer un article qu’on a publier par oublie, encore vraiment merci beaucoup pour tout.
> Bien cordialement

Oh merci beaucoup @Mkkaba ! Je suis ravi si cela a pu vous aider ! J'adore .NET MAUI, et à l'image du blogueur James Montemagno, j'espère pouvoir donner envie à de petits nouveaux sans expérience, mais qui ont de grandes idées !
A bientôt au prochain article 👋

**Comment by ahipki** [comment]
Bonjour,

Peut-être que vous savez : j'ai créé une application MAUI. J'ai créé un installeur msix. Et quand je l'installe sur mon ordinateur qui a servi au développement de l'app, cela fonctionne très bien. Par contre, je l'ai installé sur un autre ordinateur qui vient d'être mis à zéro. J'ai dû télécharger .NET Desktop Runtime et dès que j'ouvre l'application, rien ne se passe. 
J'ai tenté avec une application le plus simple possible et j'ai téléchargé au hasard .NET 8.0 SDK (V8.0.303) Windows x64 et celle-ci se lance. Par contre toujours pas pour mon application à moi. Faut-il certaines choses installées sur l'ordinateur pour pouvoir utiliser des application MAUI ?

**Reply by ahipki** [reply]
(résolu) c'était une exception non gérée pour la récupération d'un fichier sur un chemin non relatif qui n'existe pas sur l'autre poste.

**Comment by Mkkaba** [comment]
Je pense qu'il faut revoir les configurations


Le jeu. 11 juil. 2024 à 10:09, Annie Hip-Ki ***@***.***> a
écrit :

> Bonjour,
>
> Peut-être que vous savez : j'ai créé une application MAUI. J'ai créé un
> installeur msix. Et quand je l'installe sur mon ordinateur qui a servi au
> développement de l'app, cela fonctionne très bien. Par contre, je l'ai
> installé sur un autre ordinateur qui vient d'être mis à zéro. J'ai dû
> télécharger .NET Desktop Runtime et dès que j'ouvre l'application, rien ne
> se passe.
> J'ai tenté avec une application le plus simple possible et j'ai téléchargé
> au hasard .NET 8.0 SDK (V8.0.303) Windows x64 et celle-ci se lance. Par
> contre toujours pas pour mon application à moi. Faut-il certaines choses
> installées sur l'ordinateur pour pouvoir utiliser des application MAUI ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Kapusch/blog-dotnet-maui/discussions/5#discussioncomment-10019481>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2HOW44SXTXZLXD765R3QV3ZLZKTZAVCNFSM6AAAAABKWUVP7WVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBRHE2DQMI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Nom : Mohamed Kaba

Etudiant(UKAG)

Ecole polytechnique(Génie logiciel)

Tel : +224 627888569/610111627

Email ***@***.***

**Reply by ahipki** [reply]
merci, en fait c'était une exception non gérée pour la récupération d'un fichier sur un chemin non relatif qui n'existe pas sur l'autre poste.
